index.js

```diff
@@ -115,9 +106,8 @@ export function execa(rawFile, rawArgs, rawOptions) {
 		const dummySpawned = new childProcess.ChildProcess();
 		const errorPromise = Promise.reject(makeError({
 			error,
-			stdout: '',
-			stderr: '',
-			all: '',
+			stdio: Array.from({length: stdioLength}),
+			all: options.all ? '' : undefined,
```

Comment:
Small improvement. When the `all` option is `false`, `error.all` is `undefined`. However, in this specific case, it was left as an empty string instead, which was inconsistent.